src/selectors/__tests__/getPrevRank.ts

```diff
@@ -0,0 +1,20 @@
+import { initialState, reducerFlow } from '../../util'
+import { getAllChildren } from '../../selectors'
+import { newSubthought, newThought, cursorBack } from '../../reducers'
+import getPrevRank from '../getPrevRank'
+
+it('getPrevRank when all the thoughts in the context are hidden', () => {
+
+  const steps = [
+    newThought('a'),
+    newSubthought('=archive'),
+    cursorBack
+  ]
+
+  const stateNew = reducerFlow(steps)(initialState())
+
+  const children = getAllChildren(stateNew, ['a'])
```

Comment:
Use `getAllChildrenSorted`, since we want the first child to have the lowest rank.